src/schema/sources.ts

```diff
@@ -955,7 +957,11 @@ export const canPostToSquad = (
     return false;
   }
 
-  return sourceRoleRank[sourceMember.role] >= squad.memberPostingRank;
+  const memberRank = sourceRoleRank[sourceMember.role];
+
+  if (memberRank >= sourceRoleRank.moderator) return true;
```

Comment:
why is this needed specifically, does below case not solve it?

Reply:
@capJavert  We need it so moderators can bypass the moderation required check. If we remove this line, the next one will return false even if user is moderator or higher.

Reply:
I get it but it should still compare with `squad.memberPostingRank`, because if we introduce squads where only admins can post this would no longer work.

Below is more correct I would say (please check):

```suggestion
  if (memberRank >= sourceRoleRank.moderator && memberRank >= squad.memberPostingRank) {
  return true;  
}
```